### PR TITLE
Feature: new handler 

### PR DESCRIPTION
**Pull request by FacundoPucheta**
- New handler added: 'getDesignSpectrumHandler' .

### CODE REVIEW

File: src/handlers/getDesignSpectrumHandler.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const getDesignSpectrum = require("../controllers/getDesignSpectrum");
+
+const getDesignSpectrumHandler = async (req, res) => {
+  const { location, standardType, soilType } = req.body;
```
Comment (SebaSatelier) [inline src/handlers/getDesignSpectrumHandler.js]: current front-end does not send the parameters by query?

**Review by manduinca — DISMISSED** [review]
LGTM!

**Review by SebaSatelier — CHANGES_REQUESTED** [review]

File: src/handlers/getDesignSpectrumHandler.js (hunk 1 above)
Comment (FacundoPucheta) [reply src/handlers/getDesignSpectrumHandler.js]: fixed 👌🏼

**Review by SebaSatelier — APPROVED** [review]
good!

**Review by manduinca — APPROVED** [review]

**Pull request merged**